unittests/Feature/FeatureModel.cpp

```diff
@@ -48,14 +50,9 @@ TEST(FeatureModel, cloneRoot) {
 
 TEST(FeatureModel, cloneRelationship) {
   FeatureModelBuilder B;
-  BinaryFeature AA("aa");
-  BinaryFeature AB("ab");
-  auto CS = Feature::NodeSetType();
-  CS.insert(&AA);
-  CS.insert(&AB);
-  B.addParent("aa", "root")->addFeature(AA);
-  B.addParent("ab", "root")->addFeature(AB);
```

Comment:
I guess the parent relation is not strictly needed in this test or?

Reply:
As we add those features without changing root, those edges are already implied (so is the existence of a root node, which it neither of those two).